app.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories.controllers");
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
+
+app.use((err, req, res, next) => {
+  res.status(err.status).send({ msg: err.msg });
```

Comment:
This would be good to use specifically for a 500 error - I would amend this to be specific for your 500 error for the time being